test/test_loadbalancing.yml

```
@@ -0,0 +1,38 @@
+  loadbalance-hash-1:
+    image: web
+    expose:
+      - "81"
+    environment:
+      WEB_PORTS: 81
+      VIRTUAL_HOST: loadbalance-enabled.nginx-proxy.tld
+    labels:
+      com.github.nginx-proxy.nginx-proxy.loadbalance: "hash $$remote_addr;"
+
+  loadbalance-hash-2:
+    image: web
+    expose:
+      - "82"
+    environment:
+      WEB_PORTS: 82
+      VIRTUAL_HOST: loadbalance-enabled.nginx-proxy.tld

```

Rather than repeat the container, use `deploy.replicas`:

```suggestion
    deploy:
      replicas: 2
```